### ISSUE TITLE
Wrong IP in HP-Feeds Logs

### ISSUE THREAD

**Issue by simphide** [CommunityHoneyNetwork/CHN-Server#173]
Hello,

in the hp-feeds logs is only the interal docker ip not the one I set via the environment variable.
The set IP-Adress is also displayed correctly in the webinterface...

hp-feeds log:
```
timestamp="2021-01-20T07:17:17.127402Z", ssh_version="b'SSH-2.0-PuTTY_Release_0.74'", type="cowrie.sessions", sensor="6cd840a1-a208-47f5-87ad-61cd883ce935", src_port="45674", dest_port="22", transport="tcp", protocol="ssh", vendor_product="Cowrie", direction="inbound", ids_type="network", severity="high", signature="SSH login attempted on cowrie honeypot", app="cowrie", tags="location:Duesseldorf,country:DE,type:cowrie", ssh_username="asd", ssh_password="asdasdasd", src="censored by simphide", dest="172.20.0.2"
```

The matching webinterface entry with the correct ip-adress:
![image](https://user-images.githubusercontent.com/8183517/105140975-c2b58f80-5af8-11eb-8556-7ce750ef98f5.png)

Is that a known problem?

Thank you
Simon

**Comment by JesseBowling** [comment]
Hi @simphide ,

This may well be a bug...will confirm and update here. :)

**Comment by JesseBowling** [comment]
@simphide : Well, I'm not sure that setting the IP_ADDRESS config (from CHN) is the right way to get the real honeypot IP in cowrie.  That variable is only used (currently) during the registration portion with the server (so that's working as expected). That said, while exploring options for adding support for the use case to cowrie, the variables that are supposed to set the IP in the logs don't appear to be working.

Could be user error so I'm digging further on this. 

In the meantime, I have previously run containers using `host` networking with Docker which removes all the NAT/private networking magic that Docker does, but they may result in other undesired states...YMMV.

**Comment by JesseBowling** [comment]
Hi!

Confirmed that "IP_ADDRESS" was only being used (and inconsistently at that) to set the IP in the CHN-Server Sensor tab, with no other uses.

We've modified cowrie via #51 to actually use a variable "REPORTED_IP", and if present and set will use that value for both the sensor registration and the hpfeeds logs. 

Working on adding this to our other supported honeypots, and this feature should be generally available in 1.9.2. Will take some time to get all the code in place, tested, burned in, and documented. :)
